colcon.pkg

```diff
@@ -0,0 +1,8 @@
+# Configuration file for colcon (https://colcon.readthedocs.io).
```

Comment:
We also want to mention why we are adding this file here so we remember the original cause to introduce the file and listing only a subset of the versions that the sotfware can depend on.

Reply:
comments added in https://github.com/gazebosim/gz-tools/pull/142/commits/805d1fd3a50601bf9c085addcf24a5c177c82047